crates/polars-stream/src/physical_plan/lower_group_by.rs

```diff
@@ -167,6 +169,51 @@ fn try_lower_elementwise_scalar_agg_expr(
             }))
         },
 
+        #[cfg(feature = "bitwise")]
+        AExpr::Function {
+            input: inner_exprs,
+            function: IRFunctionExpr::Bitwise(inner_fn),
+            options,
+        } if matches!(
+            inner_fn,
+            IRBitwiseFunction::And | IRBitwiseFunction::Or | IRBitwiseFunction::Xor
+        ) =>
+        {
```

Comment:
You forget the following two checks on the input here:

```rust
if is_input_independent(input, expr_arena, expr_cache) {
    // TODO: we could simply return expr here, but we first need an is_scalar function, because if
    // it is not a scalar we need to return expr.implode().
    return None;
}

if !is_elementwise_rec_cached(input, expr_arena, expr_cache) {
    return None;
}
```